plugins/notification/src/index.ts

```diff
@@ -95,6 +95,16 @@ export interface NotificationTemplate {
   subjectTemplate: string
 }
 
+/**
+ * @public
+ */
+export interface NotificationPresentation {
+  title: IntlString
+  body: IntlString
+  intlParams: Record<string, string | number>
```

Comment:
should it be titleParams and bodyParams? Semantic not clear right now

Reply:
I have an idea to use intlParams for both of them. The reason for this is to be able to change the view of result notification only by changing the translations. You can easily move things from title to body back and forward and to see which options work better and this doesn't require to change where to store params to fulfil the translation